README.md

```diff
@@ -154,6 +158,9 @@ ggplot() +
 
 ### Spatial (sp) objects
 
+> ***IMPORTANT NOTE** All support for Spatial objects (`sp`) is
+> deprecated in {bcmaps} v1.2.0. Please use `sf` objects with {bcmaps}.*
+
 If you aren’t using the `sf` package and prefer the old standard
 [`sp`](https://cran.r-project.org/package=sp) way of doing things, set
 `class = "sp"` in either `get_layer` or the shortcut functions:
```

Comment:
Should we just delete this text rather than keeping it around?

Reply:
Good idea, I'll remove it all.